src/main.cpp

```diff
@@ -1166,6 +1166,15 @@ bool IsDAAEnabled(const CChainParams &chainparams, const CBlockIndex *pindexPrev
 }
 #endif
 
+bool LimitFree()
```

Comment:
"LimitFree" implies that it'll return the limit.  I'd prefer this to be called something like "AreFreeTxDisallowed".

Reply:
Good idea, but it actually it doesn't mean that exactly either...it really means, are free transactions subjet to rate limiting, so maybe something like :  "AreFreeTxnsRateLimited", will do the trick.  I'll make the fix.

Reply:
hmmm, actually it means both:  if it returns true then it means free txns are disallowed, if it returns false then it means not subject to rate limiting...so I think your AreFreeTxnsDisallowed, is more clear and closer to the truth.